libs/coin-modules/coin-stacks/src/bridge/utils/misc.ts

```diff
@@ -8,16 +8,16 @@ import {
   cvToJSON,
 } from "@stacks/transactions";
 
-import { decodeAccountId } from "../../../../account";
-import { fetchFullMempoolTxs, fetchNonce } from "../../bridge/utils/api";
+import { decodeAccountId } from "@ledgerhq/coin-framework/account/index";
+import { fetchFullMempoolTxs, fetchNonce } from "../../network/api";
 import {
   DecodedSendManyFunctionArgsCV,
   MempoolTransaction,
   StacksNetwork,
   TransactionResponse,
-} from "./api.types";
-import { getCryptoCurrencyById } from "../../../../currencies";
-import { encodeOperationId, encodeSubOperationId } from "../../../../operation";
+} from "../../network/api.types";
```

Comment:
Now that you have `index.ts` in `network` you can use it here and avoid `"../../network/api"` and `"../../network/api.types";`